modules/weko-records-ui/weko_records_ui/views.py

```
@@ -1480,9 +1481,102 @@ def dbsession_clean(exception):
+                        bucket_id=new_bucket_id).first() is not None:
+                current_app.logger.warning(
+                    'Storage API denied. reason=invalid_new_bucket, api={}, '
+                    'user_id={}, pid={}, new_bucket_id={}, new_version_id={}'.format(
+                        request.path, user_id, pid, new_bucket_id, new_version_id))
+                return denied
+    except (PIDDoesNotExistError, NoResultFound):
+        current_app.logger.warning(
+            'Storage API denied. reason=pid_not_found, api={}, user_id={}, '
+            'pid={}'.format(request.path, user_id, pid))
+        return denied
+    except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)